reference/import_export/simple-import.rst

```diff
@@ -0,0 +1,42 @@
+How import works
+================
+
+.. _Icecat: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Bundle/BatchBundle
+
+When you install an instance of Akeneo PIM, a set of data are imported: products, attributes, users, etc. For example, you can see the entities imported by the `Icecat`_ sample demo.
+
+For each entity type, as described in :doc:`/reference/import_export/main-concepts`, we import a set of entities using a Reader, a Processor and a Writer.
+
```

Comment:
These batch elements mainly use common internal API as Factory, ObjectUpdater, Validator and Saver (+ maybe a link to http://docs.akeneo.com/latest/cookbook/catalog/common/index.html )